src/main/java/com/teragrep/cfe_39/configuration/Configuration.java

```diff
@@ -43,46 +43,21 @@
  * Teragrep, the applicable Commercial License may apply to this file if you as
  * a licensee so wish it.
  */
-package com.teragrep.cfe_39;
-
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+package com.teragrep.cfe_39.configuration;
 
+import java.io.IOException;
 import java.util.Properties;
 
-import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+public interface Configuration {
+
+    ConfigurationImpl loadPropertiesFile() throws IOException;
+
+    Configuration with(String key, String value);
 
-public class ConfigTest {
+    String valueOf(String key);
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(ConfigTest.class);
+    Properties toKafkaConsumerProperties();
```

Comment:
use separate object for each of the properties

Reply:
Solved in 62ba79e2c2f5e8fdc8a2c3c02bdac68583da5c9f